src/main/java/seedu/address/commons/exceptions/DataConversionException.java

```diff
@@ -1,7 +1,7 @@
 package seedu.address.commons.exceptions;
 
 /**
- * Represents an error during conversion of data from one format to another
+ * Represents an error during loading of files from the file system.
```

Comment:
I think this belongs in the other commit, together with the name change.

Reply:
Strictly speaking, the correct order is to tweak the exception first, and then remove the catch blocks. The latter is more justifiable after the exception has been tweaked.